spec/acceptance/golden_masters_spec.rb

```diff
@@ -1,29 +1,28 @@
 require 'spec_helper'
 
 describe 'Golden masters' do
-  let(:storage)         { RetirementSwap::Storage::Memory.new }
-  let(:output)          { RetirementSwap::Output::IOWriter.new(StringIO.new) }
-  let(:retirement_swap) { RetirementSwap::SwapAlgorithm.new(storage, output) }
-
-  it 'returns the correct results for the old spacewarps data' do
-    storage = RetirementSwap::Storage::Memory.new
-    output = RetirementSwap::Output::IOWriter.new(StringIO.new)
-    processor = RetirementSwap::Processor.new(storage, output, "52c1cf443ae7407d88000001" => {"algorithm" => "swap"})
-
-    classifications = File.readlines(File.expand_path("../../fixtures/spacewarps_ouroboros_classifications.json", __FILE__))
-                          .map {|line| JSON.parse(line) }
+  let(:storage) { RetirementSwap::Storage.new(DB) }
+  let(:output) { RetirementSwap::Output::IOWriter.new(StringIO.new) }
+  let(:processor) { RetirementSwap::Processor.new(storage, output, "52c1cf443ae7407d88000001" => {"algorithm" => "swap"}) }
```

Comment:
Line is too long. [123/80]<br>Space inside { missing.<br>Space inside } missing.